rest/rest/src/main/java/io/syndesis/rest/v1/handler/integration/IntegrationSupportHandler.java

```diff
@@ -115,11 +120,18 @@ public int importModels(ModelExport export) throws IOException {
                 case Integration:
 
                     Integration integration = (Integration) model.getData();
+                    integration = new Integration.Builder()
+                        .createFrom(integration)
+                        .desiredStatus(Integration.Status.Draft)
+                        .build();
+
                     // Do we need to create it?
                     if (dataManager.fetch(Integration.class, integration.getId().get()) == null) {
-                        dataManager.create(integration);
+                        LOG.info("Creating integration: "+integration.getName());
+                        integrationHandler.create(sec, integration);
```

Comment:
Architectural wise I think it's not so nice to call a Rest endpoint from another rest endpoint as this hinders the API to evolve freely and introduces an unnecessary coupling. Instead, I would extract the common used code into a lower-layer class and use this from both, here and the endpoint.